packages/ERTP/src/paymentLedger.js

```diff
@@ -145,6 +147,18 @@ export const preparePaymentLedger = (
     { valueShape: amountShape },
   );
 
+  if (
+    recoverySetsState === 'noRecoverySets' &&
+    issuerBaggage.has('paymentRecoverySets')
+  ) {
+    // Upgrade to `'noRecoverySets'` by dropping this all at once.
+    // Depending on conditions elsewhere, this may result in a ton of
+    // payments becoming unreachable all at once. Depend on (TODO upcoming)
+    // SwingSet support for incrementalizing the resulting deallocation
+    // work.
+    issuerBaggage.delete('paymentRecoverySets');
+  }
+
   /**
    * A withdrawn live payment is associated with the recovery set of the purse
```

Comment:
Add something saying that issuers can choose to have no recovery sets, and probably to indicate that this makes sense for "payments" that don't represent the transfer of value, like price-quote payments.

Reply:
Done